okteto context create throws a nullref if $HOME/.okteto is empty or missing

**Describe the bug**
okteto context create throws a nullref if $HOME/.okteto is empty

```
$ rm -rf $HOME/.okteto
$ okteto context create

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/okteto/okteto/cmd/context.CreateCMD.func1(0x140011e2000, {0x104323e68, 0x0, 0x0})
	github.com/okteto/okteto/cmd/context/create.go:69 +0x248
github.com/spf13/cobra.(*Command).execute(0x140011e2000, {0x104323e68, 0x0, 0x0})
	github.com/spf13/cobra@v1.2.1/command.go:856 +0x668
github.com/spf13/cobra.(*Command).ExecuteC(0x140011daa00)
	github.com/spf13/cobra@v1.2.1/command.go:974 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
	github.com/okteto/okteto/main.go:126 +0x860

```
**Desktop (please complete the following information):**

- OS: macOS
- Version 1.14.10

--- a/cmd/context/create.go
+++ b/cmd/context/create.go
@@ -45,7 +45,7 @@ func CreateCMD() *cobra.Command {
 	ctxOptions := &ContextOptions{}
 	cmd := &cobra.Command{
 		Use:   "create [cluster-url]",
-		Args:  utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#create"),
+		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#create"),
 		Short: "Add a context",
 		Long: `Add a context
 

